Automattic/woocommerce-payments
# Add E2E tests for Fraud & Risk tools

Fixes #6329

#### Changes proposed in this Pull Request

Add E2E tests to validate the advanced rules from Fraud & Risk tools.

Rules tested:
- Address mismatch (Outcome: blocked)
- Purchase price threshold
  - Outside maximum limit (Outcome: blocked)
  - Outside minimum limit (Outcome: blocked)
  - Inside the rule limits (Outcome: approved)
- Order items threshold
  - Outside maximum limit (Outcome: blocked)
  - Outside minimum limit (Outcome: blocked)
  - Inside the rule limits (Outcome: approved)









#### Testing instructions

- Unit tests should be passing
- [Set up the E2E tests](https://github.com/Automattic/woocommerce-payments/tree/trunk/tests/e2e)
- Run `npm run test:e2e-dev tests/e2e/specs/wcpay/merchant/merchant-fraud-protection.spec.js` to run the new test suite
- The tests should be passing





-------------------


**Post merge**
